docs/CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Changed
 -------
 - **BACKWARD INCOMPATIBLE:** Switch ``Collection`` and ``Entity`` API viewsets
   to use Elasticsearch
+- **BACKWARD INCOMPATIBLE:** Refactor ``Relation`` model, which includes:
+
+  - renaming ``position`` to ``partiiton``
+  - renaming ``label`` to ``category`` and making it required
+  - adding ``unit``
+  - making ``collection`` field required
+  - requireing uniqe combination of ``collection`` and ``category``
+  - renaming partition's ``position`` to ``label``
+  - adding (integer) ``position`` to partition (used for sorting)
```

Comment:
Do you need to add one extra line after the sub-list?

Reply:
I would say no. Also, we don't add these extra lines in [Resolwe Bio's CHANGELOG](https://raw.githubusercontent.com/genialis/resolwe-bio/master/docs/CHANGELOG.rst).

Reply:
So, here's the catch. This works:
```
- First bullet

  - Sub-bullet 1
  - Sub-bullet 2
- Second bullet
```
But this doesn't:
```
- First bullet

  - Sub-bullet 1
  - Sub-bullet 2
  Some more text under the first bullet
```
The [standard says](http://docutils.sourceforge.net/docs/user/rst/quickref.html#bullet-lists) that a blank line is required before the first item and after the last.

Reply:
Ok, the text under the sub-bullets is a nice thing to have.

We can reformat the CHANGELOG (and the rest of the documentation, if such cases exist?) to include this extra blank line.